db/database.go

```diff
@@ -896,7 +943,7 @@ func (db *Database) UpdateAllDocChannels(doCurrentDocs bool, doImportDocs bool)
 	options := Body{"stale": false, "reduce": false}
 	if !doCurrentDocs {
 		options["endkey"] = []interface{}{true}
-		options["endkey_inclusive"] = false
+		options["endkey_inclusive"] = false  // TODO: is this valid?
```

Comment:
I asked in a [forum post](https://forums.couchbase.com/t/is-the-endkey-inclusive-view-option-still-valid/12784)